### DISCUSSION TITLE
translate Chinese documentation 翻译中文文档【常见问题解答（待更新】

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#592] [MMagic]
# translate Chinese documentation 
 翻译中文文档【常见问题解答（待更新】 

translate Chinese documentation [https://mmagic.readthedocs.io/en/latest/faq.html](https://mmagic.readthedocs.io/en/latest/faq.html)

翻译中文文档 【常见问题解答（待更新】[https://mmagic.readthedocs.io/zh_CN/latest/faq.html](https://mmagic.readthedocs.io/zh_CN/latest/faq.html) 

## Task Information

**Repository**：MMagic

**Difficulty**：Easy

**Points**：10

**Estimated Time Consumption**：7 days

## Task Status

**Claimant's Github ID**：sijiua

**Claim Time**：2023年08月01日

**PR Link**：https://github.com/open-mmlab/mmagic/pull/1967

## Points to Note

To claim the task, please reply `Claim the mission` in the comment section. After claiming the task, please add our assistant bot called MeowMewo on WeChat at: `openmmlabwx`.

After completing the task, please submit a PR with the title starting with [MMSIG]. After submission, please reply with your PR link in the comments section of this Discussion. We will review your work as soon as possible.

在评论区回复 `Claim the mission` 领取任务，领取任务后请及时添加小助手喵喵微信: `openmmlabwx`

完成任务后提交相关链接（如是PR，标题请以[MMSIG]开头）回复到本Disucssion评论区，我们会第一时间审核您的贡献。

**Comment by sijiua** [comment]
Claim the mission

**Reply by sijiua** [reply]
https://github.com/open-mmlab/mmagic/pull/1967

**Reply by crazysteeaam** [reply]
good job!
